### ISSUE TITLE
pvp personal rating differs from team

### ISSUE THREAD

**Issue by B0nisek** [SunstriderEmu/BugTracker#4272]
we are all having all games but different personal ratings. why is that?

![image](https://user-images.githubusercontent.com/22475895/94335397-af2b6980-ffdb-11ea-98d7-89706e3514ec.png)

**Comment by Ticketdudu** [comment]
I have the same problem, maybe its because leaving an arena too early, i do it sometimes when i know its a lost, and sometimes i have 1 point less than my mates after the game. But idk if that is the problem.

**Comment by Rettiy** [comment]
It's nonsense because you indeed get like 1 or 2 points less for no reason.

**Comment by B0nisek** [comment]
Yeah, I do leave a game when one of my teammates is killed and the game is lost. that never was a problem tho.

**Comment by B0nisek** [comment]
but how do I get a -16 rating from leaving 9 lost games lmao
